projects/auth0-angular/src/lib/auth.interceptor.ts

```diff
@@ -31,11 +31,11 @@ export class AuthHttpInterceptor implements HttpInterceptor {
     next: HttpHandler
   ): Observable<HttpEvent<any>> {
     const config = this.configFactory.get();
-    if (!config.httpInterceptor?.allowedList) {
+    if (!config?.httpInterceptor) {
```

Comment:
Is there any reason we should be changing this condition? I think we still want to check if allowedList is provided, as users can still use:

```
httpInterceptor: {
  allowedList: null
}
```

Reply:
I would revert thsese and other `config` related changes after restoring of` AuthClientConfig.get()`'s API

Reply:
reverted the change